pkgs/development/libraries/qt-5/5.5/qtbase/setup-hook.sh

```diff
@@ -91,7 +91,7 @@ _qtSetQmakePath() {
 
 if [ -z "$NIX_QT5_TMP" ]; then
     if [ -z "$NIX_QT_SUBMODULE" ]; then
-        NIX_QT5_TMP=$(mktemp -d)
+        NIX_QT5_TMP=$(pwd)/__nix_qt5__
```

Comment:
@avnik Please confirm that it is still possible to use nix-shell multiple times from the same directory, i.e.
```
nix-shell -A qt55.qtsvg
exit
nix-shell -A qt55.qtsvg
```
I think this will fail. You may need to add `rm -fr "$NIX_QT5_TMP"` after this line.